pkg/jetstream/config.go

```diff
@@ -51,6 +52,9 @@ type SubscriberConfig struct {
 	ConfigureStream StreamConfigurator
 	// ConfigureConsumer is a custom function that can be used to define consumer configuration from a topic.  Publisher uses it to calculate publish destination from topic.
 	ConfigureConsumer ConsumerConfigurator
+
+	// PullConsumeOption is the option that adjusts pull consume behavior
```

Comment:
Probably makes sense to name this `ConsumeOptions`

Reply:
Done